cloud.foundry.cli/src/test/java/cloud/foundry/cli/logic/apply/ApplicationRequestPlanerTest.java

```diff
@@ -137,10 +168,356 @@ void applyTest_WithRemovedObjectNotAppBeanOrAppManifestBean() {
         CfRemovedObject removedObject = new CfRemovedObject(serviceBeanMock, "propertyName", Arrays.asList("path"));
         cfChanges.add(removedObject);
 
+        ApplicationRequestsPlaner requestsPlaner = new ApplicationRequestsPlaner(appOperations);
+
         // when
         assertThrows(ApplyException.class,
-            () -> ApplicationRequestsPlaner.createApplyRequests(appOperations, appName, cfChanges));
+            () -> requestsPlaner.createApplyRequests(appName, cfChanges));
+    }
+
+
+    @Test
+    void applyTest_OnlyScalableField() {
+        // given
+        ApplicationsOperations appOperations = Mockito.mock(ApplicationsOperations.class);
+        when(appOperations.scale(any(), any(), any(), any()))
+                .thenReturn(Mono.just(mock(Void.class)));
+
+        String appName = "testApp";
+        LinkedList<CfChange> cfChanges = new LinkedList<>();
+        ApplicationBean applicationBean = new ApplicationBean();
+        ApplicationManifestBean manifestBean = new ApplicationManifestBean();
+        manifestBean.setInstances(4);
+        applicationBean.setManifest(manifestBean);
+        CfObjectValueChanged changedObject = new CfObjectValueChanged(applicationBean,
+                "instances",
+                Arrays.asList("path"),
+                "2",
+                "4");
+        cfChanges.add(changedObject);
+
+        ApplicationRequestsPlaner requestsPlaner = new ApplicationRequestsPlaner(appOperations);
+
+        // when
+        Flux<Void> requests = requestsPlaner.createApplyRequests(appName, cfChanges);
+
+        // then
+        assertThat(requests, notNullValue());
+        verify(appOperations, times(1)).scale("testApp", null, null, 4);
+        verify(appOperations, times(0)).addEnvironmentVariable(anyString(), anyString(), anyString());
+        verify(appOperations, times(0)).removeEnvironmentVariable(anyString(), anyString());
+        verify(appOperations, times(0)).bindToService(anyString(), anyString());
+        verify(appOperations, times(0)).unbindFromService(anyString(), anyString());
+        verify(appOperations, times(0)).addRoute(anyString(), anyString());
+        verify(appOperations, times(0)).removeRoute(anyString(), anyString());
```

Comment:
This code can be further simplified (this can be done for many other test cases as well):
```suggestion
        assertThat(requests, notNullValue());
        verify(appOperations, times(1)).scale("testApp", null, null, 4);
        verifyNoMoreInteractions(appOperations);
```
We should adjust this at all appropriate places in the test cases

Reply:
Didn't even know that. Will replace it. Thanks.